x-pack/plugins/security_solution/scripts/endpoint/common/endpoint_host_services.ts

```diff
@@ -143,6 +155,54 @@ export const destroyEndpointHost = async (
   ]);
 };
 
+interface CreateVmResponse {
+  vmName: string;
+}
+
+interface CreateVagrantVmOptions {
+  vmName: string;
+  cachedAgentDownload: DownloadedAgentInfo;
+  log: ToolingLog;
+}
+
+/**
+ * Creates a new VM using `vagrant`
+ */
+const createVagrantVm = async ({
+  vmName,
+  cachedAgentDownload,
+  log,
+}: CreateVagrantVmOptions): Promise<CreateVmResponse> => {
+  try {
+    await execa.command(`vagrant destroy -f`, {
+      env: {
+        VAGRANT_CWD,
+      },
+    });
+    // eslint-disable-next-line no-empty
+  } catch (e) {}
```

Comment:
Why do we need to to a force destroy here? There are no VM created at the start. No?

Reply:
if test fails then VM is not going to be destroyed, because `afterHook` is not going to be fired

Reply:
Oh right!